lib/src/toggle-group/ToggleGroup.jsx

```diff
@@ -99,17 +139,28 @@ const ToggleGroup = styled.div`
     props.margin && typeof props.margin === "object" && props.margin.bottom ? spaces[props.margin.bottom] : ""};
   margin-left: ${(props) =>
     props.margin && typeof props.margin === "object" && props.margin.left ? spaces[props.margin.left] : ""};
+  display: inline-flex;
```

Comment:
There are two values for the attribute `display`: `inline-text` and `flex`.

Reply:
Solved!